src/systems/joint_controller/JointController.cc

```diff
@@ -85,6 +95,71 @@ void JointController::Configure(const Entity &_entity,
     return;
   }
 
+  if (_sdf->HasElement("use_force_commands"))
+  {
+    this->dataPtr->useForceCommands =
+      _sdf->Get<unsigned int>("use_force_commands");
```

Comment:
This is only checking if the element `use_force_commands` exists, not if it is set to true, but code initializes the PID and prints out "Force mode ..." without checking if the parameter is true. Consider doing
```suggestion
    this->dataPtr->useForceCommands = _sdf->Get<bool>("use_force_commands", this->dataPtr->useForceCommands).first;
  if (this->dataPtr->useForceCommands)
  {
```

Reply:
Good catch! Fixed in bc7e02c.